index.html

```diff
@@ -245,35 +269,56 @@ <h1>About Me</h1>
                             </div>
                             
                     </div>
-                </section>
+                </div>
+            </section>
                 <section class="details">
                 
                     <p>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
                 
                 <form class="form-style" action="https://formspree.io/f/mnqwzknj" method="POST">
-                   
-                    <input type="text" maxlength="30" placeholder="Full Name" required>
-            
-                
-                    <input type="email" name="email" placeholder="Enter address" required>
-            
+                   <div class="form-style">
+                       <div class="input">
+                    <input type="text" maxlength="30" placeholder="First Name" required>
+                    <input type="text" maxlength="30" placeholder="Last Name" required>
```

Comment:
- [OPTIONAL ] Great job 👍🏻 However, it does not follow the Figma design closely enough, What do you think to add more `padding-top`?

- Your design:

![bad3](https://user-images.githubusercontent.com/79058364/161383960-4413e693-6859-41a3-ac02-3e9f43db4a81.png)

- figma design:

![go111111](https://user-images.githubusercontent.com/79058364/161379300-a25ab37c-7df2-4dce-a992-2f77502fafa4.png)